README.rst

```diff
@@ -33,15 +33,15 @@ Fiaasco Usage
 
 ::
     $ cd roles_basepath
-    $ molecule init template -r chrony --url https://github.com/fiaasco/molecule-cookiecutter.git
+    $ molecule init template -r chrony --url https://github.com/unifly-aero/molecule-cookiecutter.git
```

Comment:
`molecule init template` doesn't exist anymore in molecule v3.
You have to use cookiecutter: `cookiecutter https://github.com/fiaasco/molecule-cookiecutter.git`

Reply:
true, I actually use cookiecutter gh:fiaasco/molecule-cookiecutter and that works as shorthand.
I'll modify the readme